gear/identifer_lookup/src/python/identifer_app/run.py

```diff
@@ -0,0 +1,131 @@
+"""Entrypoint script for the identifer lookup app."""
+
+import logging
+import sys
+from pathlib import Path
+from typing import Dict, Optional
+
+from centers.center_group import CenterGroup
+from flywheel import Client
+from flywheel_adaptor.flywheel_proxy import FlywheelProxy
+from flywheel_gear_toolkit import GearToolkitContext
+from identifer_app.main import run
+from identifiers.database import create_session
+from identifiers.identifiers_repository import IdentifierRepository
+from identifiers.model import Identifier
+from inputs.context_parser import ConfigParseError, get_config
+from inputs.parameter_store import (ParameterError, ParameterStore,
+                                    RDSParameters)
+from outputs.errors import ErrorWriter
+
+logging.basicConfig(stream=sys.stdout, level=logging.INFO)
+log = logging.getLogger(__name__)
+
+
+def get_identifiers(rds_parameters: RDSParameters,
+                    adcid: int) -> Dict[str, Identifier]:
+    """Gets all of the Identifier objects from the identifier database using
+    the RDSParameters.
+
+    Args:
+      rds_parameters: the credentials for RDS MySQL with identifiers database
+      adcid: the center ID
+    Returns:
+      the dictionary mapping from PTID to Identifier object
+    """
+    identifiers = {}
+    identifers_session = create_session(rds_parameters)
+    with identifers_session as session:
+        identifiers_repo = IdentifierRepository(session)
+        center_identifiers = identifiers_repo.list(adc_id=adcid)
+        if not center_identifiers:
```

Comment:
Can you explain this to me? 😅
I have to be missing something, but if center_identifiers returns nothing, is it setting identifiers from nothing?